src/pages/admin/admin.tsx

```diff
@@ -0,0 +1,32 @@
+import 'tailwindcss/tailwind.css'
```

Comment:
this should be done like that that bad idea.

we should have a `global.css`
and import it on `layout.tsx`